src/components/Search/index.tsx

```diff
@@ -712,7 +712,7 @@ function Search({
 
             requestAnimationFrame(() => Navigation.navigate(ROUTES.SEARCH_REPORT.getRoute({reportID, backTo})));
         },
-        [isMobileSelectionModeEnabled, type, toggleTransaction, hash, queryJSON, handleSearch, searchKey, markReportIDAsExpense],
```

Comment:
Just want to double check that you don't remove `type` here by accident 👀

Reply:
Lint complains, not sure why in this PR though. I checked, and we don't use it in the callback.